content/server/soju.md

```diff
@@ -0,0 +1,194 @@
+---
+## Guide Information
+title: "Soju"
+description: A user-friendly IRC bouncer
+icon:
+date: 2024-06-26
+ports: [6697]
+
+## Author Information
+author: Ángel Castañeda
+email: angel@acsq.me
+matrix: "@acsquared:cyberia.club"
+
+---
+
+Soju is an irc bouncer that turns the protocol into something modern everyone can use with chat history, persistent connections, and multiserver support.
+
+This guide assumes you have a subdomain at `irc.example.org` and are port forwarding at 6697.
+
+## Installation
+
+Soju is not in debian 12's repos, so we'll need to build it from source.
+
+```sh
+VERSION=v0.8.0
+curl -fLO https://git.sr.ht/~emersion/soju/archive/$VERSION.tar.gz
```

Comment:
Soju has [moved to Codeberg](https://git.sr.ht/~emersion/soju/commit/65410474ca95e00162b9a42752017f8e49de42e7):
```suggestion
Soju is not in Debian 12's repos, so we'll need to build it from source.

```sh
VERSION=v0.8.0
curl -fLO https://codeberg.org/emersion/soju/archive/$VERSION.tar.gz
```

Reply:
knew this day would come :(